### ISSUE TITLE
Send of authentication data does not work with Firefox ESR

### ISSUE THREAD

**Issue by dbareiro** [jitsi/jicofo#155]
Hi!

I just set up user authentication based on this [1] document (Secure domain). The installation of Jitsi Meet was made with the Debian packages provided by you.

I am testing on Debian Jessie with Chromium 55.0.2883.75. Here I can send the authentication data (JID and password) and then I can enter to the conference room. But in Firefox ESR 45.6.0, the button to send the data does not seems to work.

Can you replicate this behavior?

Thanks in advance.

Kind regards,
Daniel

[1] https://github.com/jitsi/jicofo

**Comment by damencho** [comment]
If this is a problem with Firefox I suspect this is not a jicofo bug, but jitsi-meet.
Did you test with latest FF 50?

**Comment by dbareiro** [comment]
Hi, Damian.

I have tested with Firefox 50 on Windows Server 2012 and here I had no problems, so this seems to be a problem with Firefox ESR.

I'm going to add some more context information that I had not mentioned in my initial post. In my configuration I was not using a "guest" domain. So I was using authentication for everyone. I noticed yesterday (doing tests between two users using Chromium, given the existence of this problem with Firefox ESR) that all users in a conference become moderators. I had thought that adding an authentication layer would not change the previous behavior and that the moderator would be the person who created the room.

After some tests I think I've figured out how this works. Jitsi Meet considers that every authenticated user with Prosody + Jicofo is the owner (moderator) of the conference. So if I do not use a "guest" domain, all conference participants become moderators.

I have now created a "guest" domain. If the conference room does not exist, Jitsi Meet shows to the user a window notifying him/her that the moderator has not yet created the room. It also shows a button asking if s/he is the moderator. If the user presses the button, then in a new window s/he can enter username and password. I do not know why, but after having added the domain "guest", the button to send the credentials worked with Firefox ESR. 

In this scenario, any user who subsequently attempts to enter the room, would do so as anonymous and would not have moderator privileges. But I have found an unexpected effect here con Firefox ESR y Chromium.

If a user who is considered a moderator enters their credentials, they are saved without the browser asking for that action. This eventually causes that in the future a user can become a moderator of a room that was created by another user. This happens in both Firefox ESR and Chromium. Clearing the cache seems to avoid this. But the strange thing is that both browsers do not display a window asking if the user wants to save the password and, if the user saves them, this could generate the mentioned problem that I think we should try to avoid.

So I think we have two issues:

1. Unexpected cache of credentials, which could cause a conference to have more than one moderator.
2. The credentials submission button does not work in Firefox ESR if there is no "guest" domain (although this domain may be expected to exist when using authentication. if so, we could rule this issue out).

Kind regards,
Daniel
